src/components/partner/partner.resolver.ts

```diff
@@ -112,6 +117,37 @@ export class PartnerResolver {
     return await loadSecuredIds(loader, partner.countries);
   }
 
+  @ResolveField(() => SecuredLanguages)
+  async languagesOfConsulting(
+    @Parent() partner: Partner,
+    @Loader(LanguageLoader) loader: LoaderOf<LanguageLoader>,
+  ): Promise<SecuredLanguages> {
+    const languagesOfConsultingForLoader =
+      partner.languagesOfConsulting.value.map(
+        (id) =>
+          ({
+            id,
+            view: { active: true },
+          } as const),
+      );
+    const languages = (
```

Comment:
I added a helper to make this easier: `loadSecuredIds`. Check it out.

Reply:
The helper does not work in this case because language loader requires a view property. Do you know what that property is for? Should 'active' always be true? If so I could probably just make another helper

Reply:
It's for the changesets feature, which is still WIP. For this use case it'll always be the active version.
It wraps another helper: `loadManyIgnoreMissingThrowAny`. You could use that instead.
Like this:
```ts
const { value: ids, ...rest } = partner.languagesOfConsulting;
const value = await loadManyIgnoreMissingThrowAny(
  loader,
  ids.map((id) => ({ id, view: { active: true } } as const)),
);
return { ...rest, value };
```